helsingborg-stad/app-mitt-helsingborg
# fix: rename TimespanButton to TimeSpanButton across several files,

## Explain the changes you’ve made

I have renamed every usage and instance of TimespanButton to TimeSpanButton for the sake of consistency, since they were both used in different places. It also adds the missing TimeSpanButton to the molecules index file.

## Explain why these changes are made

For the sake of naming consistency when coding.

## Explain your solution

Simple rename using `git mv`

## How to test the changes?

Run the application in any mode and confirm it launches. Mac and Windows file systems are not case sensitive. If you're having trouble (specifically `molecules/index.js` complaining about TimeSpanButton) you might need to `rm -rf` the `molecules/TimespanButton` directory and then do a hard reset (`git reset --hard`) to update the indices in the file system properly.

## Was this feature tested in the following environments?
- [] Building the Application on a Android device/simulator.
